doc/users/next_whats_new/ttc_font_support.rst

```diff
@@ -0,0 +1,7 @@
+TTC font collection support
+---------------------------
+
+Fonts in a TrueType collection file (TTC) can now be added and used. Internally,
+the embedded TTF fonts are extracted and stored in the matplotlib cache
+directory. Users upgrading to this version need to rebuild the font cache for
+this feature to become effective.
```

Comment:
Maybe write how that is done? (I do not really recall myself...)

(Also one may consider having a public method that is directly importable from `matplotlib`, but that is another question...)

Reply:
You mean rebuilding the font cache? I think the simplest way to trigger a rebuild is to delete the folder printed by

```bash
python -c "import matplotlib; print(matplotlib.get_cachedir())"
```

Maybe there is a more user-friendly, programmatic way.

Reply:
Regarding the question about the public API, I'm not sure if we want to expose this. It's not directly useful for the end user. If someone wants to use a custom TTC font, they would call the font manager's `addfont()` with the path to the TTC file. This will automatically extract the TTF and add the created files to the internal list of fonts.

If a user were to use the `_split_ttc()` function, they would need to use the returned paths and add them to the font manager themself. I don't think that's what we want users to do.

Reply:
Yes, rebuilding the font cache. I seem to recall that there is a function to rebuild the cache without removing it, but cannot find it now... Googling gives `matplotlib.font_manager._rebuild()` (and quite a few questions), but that is not a current approach.

Anyway, some sort of info about how to do this (maybe it is in the docs to link to) I would think is useful since probably quite a few users like to do that.